pkg/storage/couchdb/couchdbstore.go

```diff
@@ -130,6 +130,8 @@ func (p *Provider) Close() error {
 // CouchDBStore represents a CouchDB-backed database.
 type CouchDBStore struct {
 	db *kivik.DB
+	// TODO change to leveldb
```

Comment:
I'd suggest making an issue for it and linking it here in the comment

Reply:
done